specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/job.json

```diff
@@ -0,0 +1,931 @@
+                    "description": "The time at which the job started."
+                },
+                "state": {
+                    "type": "string",
+                    "description": "The state of the job."
+                },
+                "trackingUrl": {
+                    "type": "string",
+                    "description": "The link to the web-ui for details of the job."
+                },
+                "uber": {
+                    "type": "boolean",
+                    "x-nullable": false,
+                    "description": "Whether job running in uber mode."
+                },
+                "usedMem": {
+                    "type": "integer",
+                    "format": "int64",
+                    "x-nullable": false,
+                    "description": "The amount of memory used by the job."
+                },
+                "username": {
+                    "type": "string",
+                    "description": "The userid of the person who submitted the job."
+                }
+            }
+        },
+        "Userargs": {
+            "description": "Gets or sets the object containing the user arguments.",
+            "properties": {
+                "arg": {
+                    "type": "array",
+                    "readOnly": true,
+                    "items": {
+                        "type": "string"
+                    },
+                    "description": "The list of args defined by the user."
+                },
+                "callback": {
+                    "type": "object",
+                    "description": "The callback URL, if any."
+                },
+                "define": {
+                    "type": "array",
+                    "readOnly": true,
+                    "items": {
+                        "type": "string"
+                    },
+                    "description": "The define properties defined by the user."
+                },
+                "enablelog": {
+                    "type": "string",
+                    "description": "Whether or not the user enabled logs."
+                },
+                "execute": {
+                    "type": "string",
+                    "description": "The query defined by the user."
+                },
+                "file": {
+                    "type": "object",
+                    "description": "The query file provided by the user."
+                },
+                "files": {
+                    "type": "object",
+                    "description": "The files defined by the user."
+                },
+                "jar": {
+                    "type": "string",
+                    "description": "The JAR file provided by the user."
+                },
+                "statusdir": {
+                    "type": "object",
+                    "description": "The status directory defined by the user."
+                }
+            }
+        }
+    },
+    "parameters": {

```

Comment:
Why are these parameters on the client?  Are they not expected to change across API calls?

Reply:
The behavior comes from Hyak-based Job SDK. You can refer code [here](https://github.com/Azure/azure-sdk-for-net/blob/master/src/ResourceManagement/HDInsightJob/HDInsightJob/Generated/HDInsightJobManagementClient.cs#L176-L192). The user name is used for authorization.

Reply:
I see, so this is to keep parity with the older SDK.